### PR TITLE
Zach's String + List Exercises

### PR DESCRIPTION
**Pull request by zbay**
Hello, @eah13. I've done my String and List exercises. Here's the link at my personal page: http://zbay.github.io/spring2014/2014/01/30/zbayStringsLists.html

### CODE REVIEW

**Comment by zbay** [comment]
Just noticed there's an unwanted indentation at the beginning to every code block. Looking at the raw markdown, I'm not sure why. It shouldn't affect how the code executes but it's annoying nonetheless.

**Comment by gerbal** [comment]
@zbay it is a weird css error. I've fixed it in the main silshack repo.

File: _posts/2014-01-30-zbayStringsLists.md [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+    testEqual(reverse(""),"")
+
+Output--
+
+    Pass
+    Pass
+    Pass
+
+Exercise 7:
+Used my reverse() method from exercise 6 as part of mirror()'s concatenation.
```
Comment (eah13) [inline _posts/2014-01-30-zbayStringsLists.md:34]: Remember a function is called like this: `function()`.  Methods only apply to classes of objects (like strings or lists or classes you define, which we're not going to really cover), and you do them like this: `object.method()`.

**Comment by eah13** [comment]
@zbay nice work.  You can merge in recent changes from the silshack repo to get @gerbal's awesome fix of the indentation problem.

File: _posts/2014-01-30-zbayStringsLists.md (hunk 1 above)
Comment (zbay) [inline _posts/2014-01-30-zbayStringsLists.md:34]: My bad for mixing up the terminology. Familiar with Java but still a noob at Python.

**Pull request merged**